src/pcre2_match.c

```diff
@@ -422,8 +423,13 @@ if (caseless)
         d = *p++;
         }
 
-      ur = GET_UCD(d);
-      if (c != d && c != (uint32_t)((int)d + ur->other_case))
+      if (turkish_casing && UCD_ANY_I(d))
+        {
+        c = (c == 0x0130)? 0x69 : (c == 0x49)? 0x0131 : c;
+        d = (d == 0x0130)? 0x69 : (d == 0x49)? 0x0131 : d;
+        if (c != d) return -1;  /* No match */
+        }
+      else if (c != d && c != (uint32_t)((int)d + (ur = GET_UCD(d))->other_case))
```

Comment:
This will need to be added to the JIT code too.

Reply:
making your tests `\=no_jit` could help avoiding breaking the CI.
if using a new OP it will likely work even without an implementation so it can be done at your own leisure.

Reply:
We should be careful about adding this flag, since it is easy to forget about the tests later.